test/test_sparse_prop.py

```diff
@@ -0,0 +1,66 @@
+# Owner(s): ["module: sparse"]
+#
+# Test to ensure sparsity information propagates properly into traced graph.
+#
+
+import torch
+
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+    run_tests,
+    TestCase,
+)
+
+
+# Common sparse data dtypes currently supported in torch.sparse.
+SPARSE_DTYPES = [
+    torch.float16,
+    torch.bfloat16,
+    torch.float32,
+    torch.float64,
+]
+
+# Common sparse index dtypes currently supported in torch.sparse.
+SPARSE_ITYPES = [
+    torch.int32,
+    torch.int64,
+]
+
+# Sparse layouts currently supported in torch.sparse.
+SPARSE_LAYOUTS = [
+    torch.sparse_coo,
+    torch.sparse_csr,
+    torch.sparse_csc,
+    torch.sparse_bsr,
+    torch.sparse_bsc,
+]
+
+
+class TestSparseProp(TestCase):
+    def setUp(self):
+        TestCase.setUp(self)
+
+    @parametrize("dtype", SPARSE_DTYPES)
+    @parametrize("itype", SPARSE_ITYPES)
+    @parametrize("layout", SPARSE_LAYOUTS)
+    def test_copy(self, dtype, itype, layout):
```

Comment:
Could you move this test method, say, to test/dynamo/test_misc.py as "test_clone_input" that might be more appropriate place for testing dynamo specific utils.

Reply:
sounds good